Docs/Book/Cookbook.md

```diff
@@ -409,6 +409,8 @@ The code:
     fp = numpy.zeros((1,))
     DataStructs.ConvertToNumpyArray(AllChem.GetMorganFingerprintAsBitVect(m5, 2), fp)
 
+    # Reshape to a matrix for one sample
```

Comment:
Wasn't to me and it's not what the message suggests. 
```
>>> x = np.array([1, 2, 3])
>>> x
array([1, 2, 3])
>>> (x,)
(array([1, 2, 3]),)
>>> x.reshape(1, -1)
array([[1, 2, 3]])
```
In the scikit-learn code, it will all be converted to the same by converting it into an numpy array.
```
>>> np.array((x,))
array([[1, 2, 3]])
```
We can go with your solution as well.